mlrun/model_monitoring/controller.py

```diff
@@ -286,9 +287,11 @@ def __init__(self) -> None:
         )
 
         self.model_monitoring_access_key = self._get_model_monitoring_access_key()
-        self.tsdb_connector = mlrun.model_monitoring.get_tsdb_connector(
-            project=self.project
-        )
+        self.storage_options = None
+        if not mlrun.mlconf.is_ce_mode():
+            self._initialize_v3io_configurations()
+        elif os.environ.get("MLRUN_ARTIFACT_PATH").startswith("s3://"):
```

Comment:
I think this is more elegant:
```suggestion
        elif mlconf.artifact_path.startswith("s3://"):
```

Need to add of course
```py
from mlrun.config import config as mlconf
```